osx-setup/make-release.sh

```diff
@@ -16,6 +22,20 @@ cp /usr/local/lib/libuv.dylib osx-pkg/usr/local/lib
 # just incase they want to switch networks later
 rm osx-pkg/usr/local/bin/lokinet-bootstrap
 cp ../lokinet-bootstrap osx-pkg/usr/local/bin
+#echo "Copying lokinet-control-panel.app"
+rm -fr osx-pkg/Applications/Lokinet.app
+# requires -R to keep all the symbolic links inside the App
+# https://blog.inventic.eu/2015/03/os-x-codesign-failed-bundle-format-is-ambiguous-could-be-app-or-framework/
+cp -R Lokinet.app osx-pkg/Applications
+if [ ! -f "lokinet-brand_icon-only.icns" ]; then
+  curl http://installer.lokinet.org/lokinet-brand_icon-only.icns > lokinet-brand_icon-only.icns
```

Comment:
but why?

Reply:
thought you didn't want binary files in the repo

Reply:
why http?

Reply:
because I'm not concerned about someone hacking our icon